### ISSUE TITLE
Jaws announces the zxx value of the lang attribute

### ISSUE THREAD

**Issue by joppekroon** [FreedomScientific/VFO-standards-support#691]
### Summary

When reading a span with a lang attribute set to "zxx" to indicate the text is known to be not in any language, Jaws announces "zxx" followed by the contents.

### Expected result

In a perfect world I'd expect JAWS to spell the contents, but I'd be satisfied with it not announcing "zxx".

### Actual result

JAWS announces "zxx" before it announces the contents.

### Example

```<p>Here is a list of part numbers: <span lang="zxx">9RUI34 8XOS12 3TYY85</span>.</p>```

Announces:

Here is a list of part numbers colon z x x 9roowee thirtyfour...

#### JAWS version and build number
Version 2019.1909.28 ILM

 
#### Operating System and version
Windows 10 Enterprise, Version 1803

#### Browser and version:
Reproduced both on Chrome and Edge.
Chrome 108.0.5359.125 (Official Build) (32-bit) (cohort: Stable) 
Edge 108.0.1462.76 (Official build) (64-bit)

**Comment by JAWS-test** [comment]
JAWS always announces the languages ​​for which it does not have a dictionary of pronunciation rules. That in itself makes sense. It is questionable whether this also makes sense for zxx. Since zxx is probably not known to most people, no output or an indication of the meaning of zxx would be useful
